unitTestSupportFunctions/calculateLift.m

```diff
@@ -0,0 +1,30 @@
+%% Function documentation
+%
+% Calculate lift force on a single body with no-slip boundary conditions in a rectangular domain
+%
+%               Input :
+%           FComplete : The complete force vector
+%              fldMsh : Nodes and elements for the fluid mesh
+%              homDBC : The global numbering of the DOFs where homogeneous
+%                       Dirichlet boundary conditions are applied
+%          parameters : Flow parameters
+%
+%
+%              Output :
+%                  Fy : Force (lift) on the body in y-direction
+%
+%% Function main body
+function Fy = calculateLift (FComplete, fldMsh, homDBC, parameters)
+
+    % Get node indices on the body
+    indices = getBodyNodes(fldMsh, homDBC);
+    
+    % Reshape the input forces to [Fx, Fy, Fz]
+    FComplete = -reshape(FComplete, [3, length(FComplete)/3])';
```

Comment:
These three lines of code can be condensed to Fy = sum(FComplete(2:noDimensions:end),1) * parameters.rho; which should some all forces in the y-direction given the residual force vector multiplied by the density.

Reply:
getBodyNodes return only the indices that lie on a body of interest (in this case cylinder). So we only want to sum of forces from the body of interest, not all the forces from Fcomplete in y-direction

Reply:
I guess then indices stands for the global numbering of the nodes on the body of interest, right?

Reply:
yes, exactly

Reply:
Ok. First rename **indices** to **idNodesOnBody** and then make an array **DOFsOnBody** which are basically, 

DOFsOnBody = zeros(length(idNodesOnBody),3);
for i = 1:length(DOFsOnBody)
DOFsOnBody(i,1) = 3*indices(i,1) - 2;
DOFsOnBody(i,2) = 3*indices(i,1) - 1;
DOFsOnBody(i,3) = 3*indices(i,1) ;
end

and then do,

Fy = - sum(FComplete(DOFsOnBody(2:noDimensions:end),1),1) * parameters.rho;

that should do, right?